ansible/roles/jenkins-workspace/files/clean_binary_tmp.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -ex
+
+cd ~binary_tmp/binary_tmp.git
+(echo; date) >> ~binary_tmp/clean_binary_tmp.log
+
+git fetch origin +master:master
+
+for b in $(git branch | sed /\*/d); do
+  if [ -z "$(git log -1 --since='4 weeks ago' -s $b)" ]; then
+    (git branch -D $b || true) |& tee -a ~binary_tmp/clean_binary_tmp.log
+  fi
+done
+
+git prune
```

Comment:
maybe add a ```nohup git gc >& ~binary_tmp/git_gc_`date +"%s"`.log ```?
That way we won't get an auto `gc` in the middle of a future `fetch`

Reply:
Please see below about gc.